.travis.yml

```diff
@@ -1,6 +0,0 @@
-language: node_js
```

Comment:
Same things as the auto-assign one, this has to do with the code for the travis information